packages/components/text/src/Variants.android.ts

```diff
@@ -0,0 +1,45 @@
+import { Text } from './Text';
+
+// GH #2268: Remove "as any" designations once RN Core properly supports Dynamic Type scaling
+
+export const Caption1 = Text.customize({
+  variant: 'caption1',
+} as any);
+export const Caption1Strong = Text.customize({
+  variant: 'caption1Strong',
+} as any);
+export const Caption2 = Text.customize({
+  variant: 'caption2',
+} as any);
+export const Body1 = Text.customize({
+  variant: 'body1',
+} as any);
+export const Body1Strong = Text.customize({
+  variant: 'body1Strong',
+} as any);
+export const Body2 = Text.customize({
+  variant: 'body2',
+} as any);
+export const Body2Strong = Text.customize({
+  variant: 'body2Strong',
+} as any);
+export const Subtitle1 = null; // Not supported on iOS
```

Comment:
Copying the variants over from iOS is a good rough estimate, but if we have a definitive set of tokens for Android typography, we should use those.

Reply:
Validated, the same design variants exist for android - I copied this to a common file for iOS and Android.